src/java/org/apache/cassandra/service/consensus/migration/TransactionalMigrationFromMode.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.service.consensus.migration;
+
+import org.apache.cassandra.service.consensus.TransactionalMode;
+
+public enum TransactionalMigrationFromMode
+{
+    none(null),
+    off(TransactionalMode.off),
```

Comment:
```suggestion
    // Unsafe migration from Paxos -> Accord where interoperability is not enabled at all
    off(TransactionalMode.off),
```

Reply:
This shouldn't be unsafe. This a transition from accord being off to some accord enabled mode

Reply:
OK, can you add a comment to that effect?